test/blackbox-tests/test-cases/variables-for-artifacts/cmt-cmti.t

```diff
@@ -0,0 +1,129 @@
+Test that %{cmt:...} and %{cmti:...} variables work correctly.
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.20)
+  > EOF
+
+Create a library with both .ml and .mli files:
+
+  $ cat > mylib.mli <<EOF
```

Comment:
Could you add a test where the library has the bytecode rules disabled? E.g. `(mode native)`. This should also fail to build the cmt and cmti's